atom/browser/native_window.h

```diff
@@ -156,6 +156,8 @@ class NativeWindow : public base::SupportsUserData,
   virtual void SetMenu(AtomMenuModel* menu);
   virtual void SetParentWindow(NativeWindow* parent);
   virtual void SetBrowserView(NativeBrowserView* browser_view) = 0;
```

Comment:
We can remove the `NativeWindow::SetBrowserView` implementation, and add a simple fallback for it by reusing the new API:

```js
BrowserWindow.prototype.setBrowserView = (view) => {
  this.resetBrowserViews()
  this.addBrowserView(view)
}
```

Reply:
I have tried to keep APIs separate to simplify usage in old code and new code written with new API. Even thinking about more explicit way what code can request new functionality. 
Your suggestion is good as it is let add new API without keeping legacy code and will not affect 
 existing user code. 
I have changed code in accordance with your suggestion.